src/cljc/rems/common/application_util.cljc

```diff
@@ -1,4 +1,5 @@
-(ns rems.common.application-util)
+(ns rems.common.application-util
+  (:require [rems.common.util :refer [build-index]]))
```

Comment:
unused?

Reply:
Yeah I ended up removing a helper that used this.